test/extended/security/scc.go

```diff
@@ -339,3 +537,100 @@ func runAsRootPSPSSR() *securityv1.PodSecurityPolicySelfSubjectReview {
 		},
 	}
 }
+
+func createPodAdminRoleOrDie(ctx context.Context, oc *exutil.CLI, sa *corev1.ServiceAccount) {
+	framework.Logf("Creating role")
+	rule := rbacv1helpers.NewRule("create", "update").Groups("").Resources("pods", "pods/exec").RuleOrDie()
```

Comment:
aren't SAs capable of performing these in their namespace by default?

Reply:
Unless there's a default role for every SA. By running the test with freshly provisioned SA I had to add these to get the test going.